aiohttp/http_parser.py

```diff
@@ -29,7 +29,7 @@
     'RawRequestMessage', 'RawResponseMessage')
 
 ASCIISET = set(string.printable)
-METHRE = re.compile('[A-Z0-9$-_.]+')
+METHRE = re.compile("[!#$%&'*+\-.^_`|~0-9A-Za-z]+")
```

Comment:
I think It might make sense to put a comment with that rule somewhere nearby:
```
#     method = token
#     tchar = "!" / "#" / "$" / "%" / "&" / "'" / "*" / "+" / "-" / "." /
#             "^" / "_" / "`" / "|" / "~" / DIGIT / ALPHA
#     token = 1*tchar
```

Reply:
Yeah, I know about blame. But it's visually more convenient to see it next to the code. Like it's done in yarl, for example.